### PR TITLE
Add delivery-instructions data class

### PR DESCRIPTION
**Pull request by pdehaan**
See https://monitor.firefox.com/breach-details/Weee

# References: 
Jira: MNTOR-
Figma: 

# Description

# Screenshot (if applicable)

Not applicable.

# How to test

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: locales/en/data-classes.ftl [hunk 1]
```diff
@@ -34,6 +34,7 @@ customer-interactions = Customer interactions
 dates-of-birth = Dates of birth
 deceased-date = Deceased date
 deceased-statuses = Deceased statuses
+delivery-instructions = Delivery instructions
```
Comment (flodolo) [inline locales/en/data-classes.ftl:37]: What is this? Delivery instructions for drivers (e.g. on Amazon)?

Can we have a comment to give context to localizers?

Comment (pdehaan) [reply locales/en/data-classes.ftl:37]: https://monitor.firefox.com/breach-details/Weee is all I know. Let me see if I can fetch the description from HIBP site.

Comment (pdehaan) [reply locales/en/data-classes.ftl:37]: > _"In February 2023, *data belonging to the Asian and Hispanic food delivery service Weee appeared on a popular hacking forum*. Dating back to mid-2022, the data included 1.1M unique email addresses from 11M rows of orders containing names, phone numbers and delivery instructions."_

Comment (flodolo) [reply locales/en/data-classes.ftl:37]: OK. Then something like

```
# Instructions that you can set up in online shopping services to get your orders 
# delivered (e.g. deliver to a specific neighbor, entrance location, etc.). 
```

**Review by flodolo — APPROVED** [review]

**Pull request merged**